### ISSUE TITLE
Other element would receive a click error when attempting to select a value in dropdown if width is smaller than value width

### ISSUE THREAD

**Issue by lyoff** [SeleniumHQ/selenium#1479]
On Selenium version 2.49.0

Error:

```
org.openqa.selenium.WebDriverException: Element is not clickable at point (70, 15). Other element would receive the click: <body></body>
```

Test:

```java
    @Test
    public void selectTest() {
        driver.get(baseUrl);
        new Select(driver.findElement(By.id("xyz"))).selectByValue("c");
    }
```

HTML

```html
<html lang="en">
<head>
</head>
<body>
<select id="xyz" style="width: 50px;">
    <option value="a">abc</option>
    <option value="b">some long text here</option>
    <option value="c">my option</option>
</select>
</body>
</html>
```

**Comment by dbenoit** [comment]
You need to but this in a loop to get the child elements of option. 

For example python code>
 i = 1
options = 3  # or supply the path to the childElementCount of option
while i <=options:
	 items = driver.find_element_by_css_selector(The Path to+">option:nth-of-type(("+str(i)+")")
	 print items.get_attribute("value"), items.text
 i=i+1
 Output:
a abc
b some long text here
c my option

**Comment by vveliev** [comment]
I have noticed that scroll to the element does not working correctly, and same error happens. 

in particular we have menu div on the top of the page, while focus does scroll page but locate element from the most top of the page (keeping it behind menu bar).

**Comment by thomasbecker** [comment]
Same issue happens to me since we upgraded to Selenium 2.49.0 to have firefox 43.x support. The button to be clicked doesn't move around, it's definitely clickable and it's visible.

Rolling back to previous selenium and firefox versions for now.

**Comment by nixsec** [comment]
I'm seeing the same issue as well. Until it's resolved, we've rolled back to the previous version.

**Comment by pataky** [comment]
The same here. We had to roll back to a previous version also.

**Comment by dmansfield** [comment]
I'm also seeing an issue where latest WebDriver (2.49.0) is clicking in the wrong location for a stationary element.  Click seems to be just below the bottom of the given element instead of in the middle.  X coordinate is fine.

Using with firefox version: firefox-38.5.0-2.el5.centos

In my scenario, the element is an anchor tag, nothing to do with select or option.

**Comment by dmansfield** [comment]
I'm guessing commit a2401418b8f73753f3fc9a7ef666be715177b6a5 is triggering this, but it still begs the question why the click coordinates are "off" in the y dimension by element.height/2.

Unless I mis-remember that an element click is supposed to be in the middle of the element.

**Comment by barancev** [comment]
Fixed by b21a4f52d37588d2938aace2f47c93d442cd052e
